cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -537,10 +537,78 @@ function Add-Path($OrigPathToAdd) {
     $NewPath = $PathToAdd + $OldPath
     # We use -Force here to make the value already existing not be an error
     $Item | New-ItemProperty -Name $PropertyName -Value $NewPath -PropertyType String -Force | Out-Null
+    # Notify running processes that they should update their environment variables (in particular, explorer.exe)
+    # This ensures we only need a shell restart, rather than a system restart
+    Update-EnvironmentVariables
     return $true
   }
 }
 
+<#
```

Comment:
This is great, thank you! In addition to the license, maybe let's also add the URL and version here. We can add it when we add the snapshots.